### ISSUE TITLE
bind:text doesn't working inside else block

### ISSUE THREAD

**Issue by RedHatter** [halfnelson/svelte-native#109]
When a `textField` is inside of an `{:else}` block the `bind:text` doesn't work. The compiler throws `ValidationError: 'text' is not a valid binding (9:15)`.

For example,  the following fails.
```
<script>
  let username = ''
</script>

<page>
  {#if true}
    <label>True</label>
  {:else}
    <textField bind:text={username} hint="Email or Username" />
  {/if}
</page>
```
But if you switch the `textField` and `label` it works fine.
```
<script>
  let username = ''
</script>

<page>
  {#if true}
    <textField bind:text={username} hint="Email or Username" />
  {:else}
    <label>True</label>
  {/if}
</page>
```

This might be related to #24. I tired removing my node_modules and reinstalling but it didn't help.

**Comment by halfnelson** [comment]
it certainly sounds like `svelte-native-preprocessor` isn't running, unless it is having trouble traversing the AST, could you please check your webpack.config.js to see if it is setup to run the preprocessor?

**Comment by RedHatter** [comment]
I'm using the [svelte-native-template](https://github.com/halfnelson/svelte-native-template) and I haven't changed any configuration so `svelte-native-preprocessor` should be running.

If the processor wasn't running the the binding should work no matter where the `textField` is, right? The fact that it only fails when inside of an else block points to it being an issue with AST traversal.

**Comment by halfnelson** [comment]
Well spotted. It was the AST traversal, I switched to using svelte's internal walk, and that fixes the issue. I have published svelte-native-preprocessor 0.1.6 which contains the fix.
